controllers/authControllers.js

```diff
@@ -27,8 +29,42 @@ try {
     "sceptersecretkey",
     {expiresIn:"2h"}
     );
+
     res.status(200).json({user, accessToken});
 } catch (err) {
+    console.log("registerUser: " + err);
+    res.status(500).json(err);
+}
```

Comment:
create another controller for this request

Reply:
Done. Created another controller.